src/main/java/com/weather/model/Weather.java

```diff
@@ -1,7 +1,17 @@
 package com.weather.model;
 
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+
+@Entity
 public class Weather extends WeatherEntry{
 
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    private int id;
+
     private String name;
```

Comment:
Use lombok library to remove boiler plate code (like getter/setter).

- [ ] use lombok library

- [ ] .

Reply:
Yes I will do it.